.textlintrc.json

```diff
@@ -83,10 +83,12 @@
         "YouTube",
         "Zend",
         "Zipkin",
+        [".net", ".NET"],
         ["3rd[- ]party", "third-party"],
-        ["bug[- ]fix(es)?", "bugfix$1"],
         ["cpp", "C++"],
-        ["dotnet", ".NET"],
+        ["[Dd]ot[Nn]et", ".NET"],
+        ["EBPF", ".eBPF"],
+        ["eb[Pp][Ff]", ".eBPF"],
```

Comment:
As far as I understand the "documentation" for textlint-rule-terminology those regular expressions are case insensitive: https://github.com/sapegin/textlint-rule-terminology (see "        // RegExp (case-insensitive) → replacement" in the sample config

You should get the same result by just putting "eBPF" in the list.

Reply:
Oh, that's even better.